### PR TITLE
Add program ids and submitter names to manual page

### PR DESCRIPTION
**Pull request by mcoughlin**
**Does this pull request make any changes to the database?**
No

Code changes that affect the database require special attention for data migration.
If your change affects the database, describe the migration plan here.

### CODE REVIEW

**Comment by mcoughlin** [comment]
This is a request of Eric Bellm.

File: growth/too/views.py [hunk 1]
```diff
@@ -564,6 +564,16 @@ class PlanManualForm(ModelForm):
         validators=[validators.DataRequired()],
         default='REPLACE ME')
 
+    submitter = TextField(
+        validators=[validators.DataRequired()],
```
Comment (lpsinger) [inline growth/too/views.py]: Is this supposed to be the user's name? If so, we can get that from `current_user`.

File: growth/too/views.py [hunk 2]
```diff
@@ -564,6 +564,16 @@ class PlanManualForm(ModelForm):
...
+        default='REPLACE ME')
+
+    subprogram_name = TextField(
```
Comment (lpsinger) [inline growth/too/views.py]: I don't understand what this field is for. Perhaps some help text?

**Comment by lpsinger** [comment]
Can you please explain why we want to be able to set the program ID?

File: growth/too/views.py (hunk 1 above)
Comment (mcoughlin) [reply growth/too/views.py]: Yup!

File: growth/too/views.py [hunk 3]
```diff
@@ -564,6 +564,16 @@ class PlanManualForm(ModelForm):
...
+        validators=[validators.DataRequired()],
+        default='GW')
+
+    program_id = SelectField(default='2')
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply growth/too/views.py]: Flask prefers textual values in their SelectField class, seems cleanest.

File: growth/too/views.py (hunk 2 above)
Comment (mcoughlin) [reply growth/too/views.py]: Added.

**Comment by mcoughlin** [comment]
Program ID is request of E  Bellm, but some of the ToOs can be Caltech only, so it makes sense.

File: growth/too/views.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (lpsinger) [reply growth/too/views.py]: What does the user see? Do they see a dropdown with numbers in it, or a dropdown that has options like "Partnership" and "Caltech"?

File: growth/too/views.py (hunk 1 above)
Comment (mcoughlin) [reply growth/too/views.py]: Yes its gone. see updated request.

File: growth/too/views.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (mcoughlin) [reply growth/too/views.py]: Fixed to use Partnership and Caltech

[the remark this answers is not in the document]
Comment (lpsinger) [reply growth/too/views.py]: Where do the program IDs and their textual names come from? Why doesn't the scheduler API take the textual values?

[the remark this answers is not in the document]
Comment (mcoughlin) [reply growth/too/views.py]: No it just speaks 1, 2, and 3. But at least we know what they mean ;)

**Comment by lpsinger** [comment]
Please pull master and rebase to pick up Travis CI fixes.

File: growth/too/views.py [hunk 4]
```diff
@@ -590,6 +596,7 @@ def plan_manual():
     form = PlanManualForm()
     form.telescope.choices = [
         (row.telescope,) * 2 for row in models.Telescope.query]
+    form.program_id.choices = [('Partnership',) * 2, ('Caltech',) * 2]
```
Comment (lpsinger) [inline growth/too/views.py]: Can this be put int he `PlanManualForm` class definition?

File: growth/too/views.py [hunk 5]
```diff
@@ -989,8 +1002,8 @@ def get_json_data_manual(form):
                       'dec': field.dec,
                       'filter_id': filter_id,
                       'exposure_time': exposure_time,
-                      'program_pi': program_pis[telescope],
-                      'subprogram_name': "ToO_manual"
+                      'program_pi': program_pis[telescope] + '/' + username,
+                      'subprogram_name': "ToO_" + subprogram_name
```
Comment (shreyasahasram08) [inline growth/too/views.py]: Could you please add the subprogram name choices underneath (like for the filters), i.e. GW, GRB, Neutrino?

File: growth/too/templates/plan_manual.html [hunk 6]
```diff
@@ -58,10 +62,14 @@
                 </div>
             </div>
             <div class=form-row>
-                <div class="form-group col-sm-12">
+                <div class="form-group col-sm-6">
                     <label for=plan_name>Queue name</label>
                     {{ render_field(form.queue_name, placeholder='Queue name') }}
                 </div>
+                <div class="form-group col-sm-6">
+                    <label for=subprogram_name>Subprogram name</label>
```
Comment (shreyasahasram08) [inline growth/too/templates/plan_manual.html]: Would you mind adding the subprogram name choices underneath (like for the filters), i.e. GW, GRB, Neutrino, just to clarify what this field is supposed to be?

File: growth/too/templates/plan_manual.html [hunk 7]
```diff
@@ -18,6 +18,10 @@
                     <label for=telescope class=form-label>Telescope</label>
                     {{ render_field(form.telescope, class='custom-select') }}
                 </div>
+                <div class="form-group col-sm-6">
+                    <label for=program_id class=form-label>Program ID</label>
+                    {{ render_field(form.program_id, class='custom-select') }}
```
Comment (shreyasahasram08) [inline growth/too/templates/plan_manual.html]: Could you add a note here that program ID is '(ZTF only)'?

**Comment by shreyasahasram08** [comment]
Aside from a few minor points, I pulled this one up and it looks good to me.

**Pull request merged**